packages/react-app/src/CreateInk.js

```diff
@@ -604,6 +614,11 @@ if (props.mode === "edit") {
     </Row>
     <Row style={{ width: "40vmin", margin: "0 auto", marginTop:"1vh", justifyContent:'center'}}>
         <Space>
+        <Col span={4}>
+          <Button
```

Comment:
I think maybe makes sense to put this next to the "save as draft" button, rather than with the drawing tools?